src/pages/careers/_dept-layout-components/_zoho-job.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { OpenPositionsProps } from './_dept-layout.types'
+import RightArrow from 'images/svg/career-right-arrow.svg'
+
+type DataProps = {
+    title: OpenPositionsProps['title']
+}
+
+const ZohoJob = (data: DataProps) => {
+    const [scriptLoaded, setScriptLoaded] = useState(false)
+
+    useEffect(() => {
+        const careerScript = document.createElement('script')
+        careerScript.src =
+            'https://js.zohostatic.com/recruit/embed_careers_site/javascript/v1.0/embed_jobs.js'
+        careerScript.addEventListener('load', () => setScriptLoaded(true))
+        document.body.appendChild(careerScript)
+    }, [])
+
+    useEffect(() => {
+        if (!scriptLoaded) return
+        addScriptForZoho()
+    }, [scriptLoaded])
+
+    const addScriptForZoho = () => {
+        const addScript = (settings) => {
+            const script = document.createElement('script')
+            const { async, text, src, id } = settings
+
+            if (async) script.async = settings['async']
+            if (text) script.text = settings['text']
+            if (src) script.src = settings['src']
+            if (id) script.id = settings['id']
+            document.head.appendChild(script)
+        }
+
+        addScript({
+            text: `
+            window.rec_embed_js && window.rec_embed_js.load({
+				widget_id:"rec_job_listing_div",
+				page_name: "${data.title}",
+				source:"CareerSite",
+				site:"https://deriv.zohorecruit.com",
+				empty_job_msg:"Sorry, there are currently no open positions."
+			});`,
+        })
+    }
+
+    const ZohoWrapper = styled.div`
+        font-family: 'IBM Plex Sans', sans-serif !important;
+        width: 100%;
+
+        .rec-group .rec-job-info:nth-child(n + 5) {
+            display: none !important;
+        }
+        #rec_job_listing_div:not(#rec_job_listing_div *) {
+            text-align: center;
+            color: var(--color-grey-5);
+            font-weight: bold;
+            font-size: 2.4rem;
+            line-height: 3.6rem;
+            width: 100%;
+            display: block;
+        }
+        .rec_job_listing_div_jobs {
+            text-align: left;
+        }
+        ul.rec-job-info {
+            position: relative;
+            margin-bottom: 0 !important;
+            padding: 24px 35px 24px 18px;
+            width: 100%;
+            display: flex;
+            border-bottom: 1px solid #f2f3f4 !important;
+            align-items: center;
+
+            &::after {
+                position: absolute;
+                content: url(${RightArrow});
+                top: 50%;
+                right: 0;
+                font-size: 16px;
+                transform: translateY(-50%);
+            }
+        }
+        .embed_jobs_with_style .rec-group li.rec-job-title {
+            width: 50%;
+            max-width: 500px;
+            margin-bottom: 0;
+            padding-right: 5px;
```

Comment:
```suggestion
            min-width: 50%;
            width: auto;
            max-width: 800px;
            margin-bottom: 0;
            padding-right: 24px;
```

Reply:
min-width: 50%; breaks the layout. Sticking with the width:50%.